README.rst

```diff
@@ -69,6 +69,20 @@ FogLAMP is currently based on C/C++ and Python code. The packages needed to buil
 - sqlite3
 - libsqlite3-dev
 
+
+Linux distributions
+-------------------
+
+FogLAMP can be build or installed in one of the following Linux distributions :
+
+- Ubuntu
```

Comment:
Best to call out Ubuntu 16 and Ubuntu 18 explicitly

Reply:
done